source/site-administrators/studio/site-policy-configuration.rst

```diff
@@ -0,0 +1,71 @@
+:is-up-to-date: True
+
+.. index:: Site Policy Configuration
+
+.. _site-policy-configuration:
+
+#########################
+Site Policy Configuration
+#########################
+
+The site policy configuration file allows the user to configure conditions for content being added to the site, such as
```

Comment:
Please enumerate what's possible now.
* Filename allowed patterns and automatic renaming rules
* File size limits
* MIME-type limits
* Content-type limits